lib/diagnostic.md

```diff
@@ -37,7 +37,7 @@ Give two examples of jQuery getter methods and two examples of jQuery setter met
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is a set of conventions for implementing html on the web. It has APIs in browsers to correct some basic coding mistakes so that the browsera know what to show when the code is wrong.
 <!-- your answer ends here -->
```

Comment:
The Document Object Model (DOM) is an application programming interface (API) for valid HTML and well-formed XML documents. It defines the logical structure of documents and the way a document is accessed and manipulated.